modules/installation-disk-partitioning.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * installing-vsphere.adoc
+// * installing-vsphere-network-customizations.adoc
+// * installing-restricted-networks-vsphere.adoc
+
+// This content was sourced from the bare metal RHCOS assembly file `modules/installation-user-infra-machines-advanced.adoc` under the `== Disk partitioning` subheader. "Disk partioning" content in the bare metal assembly is not modularized, so anything in this vsphere module should be checked against that file for consistency until such time that the large bare metal assembly can be modularized.
+
+[id="installation-disk-partitioning_{context}"]
+= Disk partitioning
+
+In most cases, data partitions are originally created by installing {op-system}, rather than by installing another operating system. In such cases, the {product-title} installer should be allowed to configure your disk partitions.
+
+However, there are two cases where you might want to intervene to override the default partitioning when installing an
+{product-title} node:
+
+* Create separate partitions: For greenfield installations on an empty
+disk, you might want to add separate storage to a partition. This is
+officially supported for making `/var` or a subdirectory of `/var`, such as `/var/lib/etcd`, a separate partition, but not both.
++
+Kubernetes supports only two filesystem partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
```

Comment:
In the bare metal version, this statement is in an "Important" call-out. It's fine as is IMO, but I wanted to point it out for consistency.

Reply:
Wow, good catch! Resolved. 🙇